website/docs/release_notes.md

```diff
@@ -8,6 +8,79 @@ import ReactPlayer from 'react-player'
 
 # Release notes
 
+## [v1.1.3](https://github.com/VirtusLab/scala-cli/releases/tag/v1.1.3)
+
+### Support for LTS Scala version aliases
+It is now possible to use `lts` and `3.lts` as Scala version aliases in Scala CLI. 
+They refer to the latest LTS version of Scala (the `3.3.x` line at the time of this release).`
+
+```bash
+scala-cli run -S lts --with-compiler -e 'println(dotty.tools.dotc.config.Properties.simpleVersionString)'
+# Compiling project (Scala 3.3.1, JVM (17))
+# Compiled project (Scala 3.3.1, JVM (17))
+# 3.3.1
+```
+
+Using the `2.lts`, `2.13.lts` & `2.12.lts` aliases returns a meaningful error, too.
+
+```bash fail
+scala-cli run -S 2.lts -e 'println(scala.util.Properties.versionString)'                                 
+# [error]  Invalid Scala version: 2.lts. There is no official LTS version for Scala 2.
+# You can only choose one of the 3.x, 2.13.x, and 2.12.x. versions.
+# The latest supported stable versions are 2.12.18, 2.13.12, 3.3.1.
+# In addition, you can request compilation with the last nightly versions of Scala,
+# by passing the 2.nightly, 2.12.nightly, 2.13.nightly, or 3.nightly arguments.
+# Specific Scala 2 or Scala 3 nightly versions are also accepted.
+# You can also request the latest Scala 3 LTS by passing lts or 3.lts.
+```
+
+Added by [@kasiaMarek](https://github.com/kasiaMarek) in [#2710](https://github.com/VirtusLab/scala-cli/pull/2710)
+
+### `--semanticdb-targetroot` and `--semanticdb-sourceroot` options
+It is now possible to set the SemanticDB target root and source root directories with unified syntax,
+independent of the target Scala and/or Java versions.
+
+<ChainedSnippets>
+
+```scala title=src/semanticdb-example.sc
+println("SemanticDB targetroot gets set to ./targetRootDir, while sourceroot gets set to the current working directory.")
+```
```

Comment:
Yeah... it won't render the `title` tag...
I need to provide something for the docs tests to check the snippet, however.
Let me try to make it clearer then

Reply:
I got rid of the `ChainedSnippets` here and described it in a (hopefully) clearer manner.
I also skip the script example in the github notes completely, it's strictly necessary here, so that `docs-tests` can verify the snippet on the CI.

Reply:
Thanks, that's far better for me.